src/NNCache.cpp

```diff
@@ -77,7 +77,13 @@ void NNCache::set_size_from_playouts(int max_playouts) {
     // cache hits are generally from last several moves so setting cache
     // size based on playouts increases the hit rate while balancing memory
     // usage for low playout instances. 150'000 cache entries is ~225 MB
-    auto max_size = std::min(150'000, std::max(6'000, 3 * max_playouts));
+    constexpr auto num_cache_moves = 3;
+    auto max_playouts_per_move =
+        std::min(max_playouts,
+            std::numeric_limits<decltype(max_playouts)>::max() /
+            num_cache_moves);
```

Comment:
I think, following the reasoning, this should be / 2*num_cache_moves (because /2 is the real max playouts). But it makes no difference as it's capped below anyway.